core/lib/spree/testing_support/kernel.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
+  module TestingSupport
+    module Kernel
+
+      private
+
+      def silence_stream(stream)
+        old_stream = stream.dup
```

Comment:
@priyank-gupta @mauazua It is only added for module Spree::TestingSupport::Kernel, so it won't be a problem outside specs. I think it could stay, as long as it's not causing any problems with circleci.